doc/modules/changes.h

```diff
@@ -6,9 +6,15 @@
  *
  *
  * <ol>
- * <li> New: The tolerance of the preconditioners of the A and S block 
- * are now available as parameters in the prm file. There is now also 
- * a section added to the manual on how to use these parameters to 
+ * <li> Fixed: Whenever the base models used by either the "depth dependent"
+ * or "averaging" material models depended on anything that requires accessing
+ * the simulator, then this led segmentation faults. This is now fixed.
```

Comment:
Hi Wolfgang and Timo,

The segmentation fault problem has been solved. Thanks!

Shangxin
On Nov 25, 2015 9:14 PM, "Timo Heister" <notifications@github.com> wrote:

> In doc/modules/changes.h
> <https://github.com/geodynamics/aspect/pull/673#discussion_r45938530>:
>
> > @@ -6,9 +6,15 @@
> >   *
> >   *
> >   * <ol>
> > - * <li> New: The tolerance of the preconditioners of the A and S block
> > - * are now available as parameters in the prm file. There is now also
> > - * a section added to the manual on how to use these parameters to
> > + * <li> Fixed: Whenever the base models used by either the "depth dependent"
> > + * or "averaging" material models depended on anything that requires accessing
> > + * the simulator, then this led segmentation faults. This is now fixed.
>
> "led to"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/aspect/pull/673/files#r45938530>.
>